libs/shared/src/entities/organization/organization.interface.ts

```diff
@@ -2,26 +2,24 @@ import { ApiServiceLevelEnum } from '../../types';
 import { IUserEntity } from '../user';
 import { MemberRoleEnum } from './member.enum';
 import { IMemberInvite, MemberStatusEnum } from './member.interface';
+import { ProductUseCases } from '../../dto';
 
 export interface IOrganizationEntity {
   _id: string;
   name: string;
   apiServiceLevel?: ApiServiceLevelEnum;
-  members: {
-    _id: string;
-    _userId?: string;
-    user?: Pick<IUserEntity, 'firstName' | '_id' | 'lastName' | 'email'>;
-    roles: MemberRoleEnum[];
-    invite?: IMemberInvite;
-    memberStatus: MemberStatusEnum;
-  }[];
   branding?: {
     color: string;
     logo: string;
-    fontColor: string;
-    fontFamily: string;
-    contentBackground: string;
-    direction: 'ltr' | 'rtl';
+    fontColor?: string;
+    fontFamily?: string;
+    contentBackground?: string;
+    direction?: 'ltr' | 'rtl';
   };
   defaultLocale?: string;
+  jobTitle?: string;
```

Comment:
I think this should be on the user entity?  Or there is some reason for it on the organization?

Reply:
No, you're right it was my mistake putting it there, will refactor it shortly, thanks for highlighting it!